doc/src/site/sphinx/connectors/cassandra_connector.rst

```diff
@@ -50,4 +50,23 @@ Cassandra native functions
 
 - now()
 - dateOf(<date>)
-- toDate(<date>)
\ No newline at end of file
+- toDate(<date>)
+
+
+Import existing tables into Crossdata
+*************************************
+To import existing tables into the Crossdata Catalog, execute this query::
+
+         IMPORT TABLES
+         USING com.stratio.crossdata.connector.cassandra
+         OPTIONS (
+          cluster "MuClusterName",
+          keyspace "MyKeyspace",
+          table "MyTable",
+          spark_cassandra_connection_host "localhost"
+          )
+
+
+Where:
+- keyspace (Optional): The Cassandra Keyspace name to import, if is omitted, Crossdata will import all keyspaces in the cluster.
```

Comment:
"The Cassandra Keyspace name to import, if is omitted, Crossdata will import all keyspaces in the cluster" could be replaced by "The Cassandra keyspace to import; when omitted, Crossdata will import all keyspaces in the cluster"